### PR TITLE
Content update 2021

### PR DESCRIPTION
**Pull request by Wint3rmute**

### CODE REVIEW

File: content/about.md [hunk 1]
```diff
@@ -28,49 +28,56 @@ cover:
 
 ### Work-related interests:
 
-- Currently designing a commercial cubesat platform at [SatRevolution](https://satrevolution.com/)
-- Playing around with UAVs - MavLink, ArduPilot & DroneKit
-- Vice president of the [Academic Aviation Club](http://akl.pwr.edu.pl)
-- Writing automation & testing scripts mainly in Python & Bash
+- Currently designing & operating a commercial CubeSat constellation @[SatRevolution](https://satrevolution.com/)
```
Comment (Oldzii) [inline content/about.md:31]: bez 'a'

File: content/about.md [hunk 2]
```diff
@@ -28,49 +28,56 @@ cover:
...
+- Playing around with UAVs - ArduPilot, computer vision, telemetry systems
+- Tech lead of the [Academic Aviation Club](http://akl.pwr.edu.pl) (2nd place [Sae Aero Design Advanced](https://www.saeaerodesign.com/) winner)
```
Comment (Oldzii) [inline content/about.md:33]: <3

File: content/about.md [hunk 3]
```diff
@@ -28,49 +28,56 @@ cover:
...
-- Web services and web apps with Flask and Django
+- Solid understanding of mission operations & telemetry system design
+- Python + Rust + Bash = ❤️
 
 ### Other interests:
 
 - Hard sci-fi
-- Lots of music, weird sounds
-- Random philosophical ideas
+- Overly pretentious music
```
Comment (Oldzii) [inline content/about.md:41]: meh

File: content/about.md [hunk 4]
```diff
@@ -28,49 +28,56 @@ cover:
...
+## CubeSats
 
-Since 2019, I've been rolling with the [Academic Aviation Club](http://akl.pwr.edu.pl) - a student club from my uni,
-dedicated to developing autonomous airplanes and drones. I've got the know-how to develop on top of ArduPilot-based flight controllers.
+During the summer of 2020 I've started working at [SatRevolution](https://satrevolution.com/) - company
+designing a cubesat satellite fleet for commercial and research usage.
 
-As a confimation of my skills: I've attended [SAE Aero Design 2020](https://www.saeaerodesign.com/) and my team has placed second in the Advance cathegory.
+I currently lead the mission operations software team -- that's where we design
+the communication & telemetry systems of our satellites, create the tools used for automated
```
Comment (Oldzii) [inline content/about.md:53]: for our

File: content/about.md [hunk 5]
```diff
@@ -28,49 +28,56 @@ cover:
...
+testing and finally talk with the satellites on orbit.
```
Comment (Oldzii) [inline content/about.md:54]: hmm create the tools used for automated testing, by this I understand that u create software for automation, not that you are using them to automate your tests?
'talk to' sounds better

File: content/about.md [hunk 6]
```diff
@@ -90,10 +97,10 @@ Unity and the server software in Django.
 
 ## The future?
 
-I've started programming with C/C++, then switched to high-level languages and technologies.
-My plan is to go full circle and return to low-level - probably with Rust. I'll try to combine
-my Linux knowledge and low-level programming (backed by high-level prototyping and testing),
-to develop on embedded devices or to build high-performance server/desktop apps.
+I've learned the basics of programming with C, then switched to high-level languages and technologies.
```
Comment (Oldzii) [inline content/about.md:100]: 'in C' here
'higher-level'?

File: content/about.md [hunk 7]
```diff
@@ -90,10 +97,10 @@ Unity and the server software in Django.
...
+My current self development plan is built around Rust. I'm aiming to combine
```
Comment (Oldzii) [inline content/about.md:101]: sounds strange, maybe 'is focused on Rust; I aim at combining

File: content/non_programming.md [hunk 8]
```diff
@@ -29,34 +29,23 @@ cover:
 I've been through a music school, learning piano for 6 years (also boosting my typing speed, which came in handy later).
 I spend most of my time with music, exploring interesting genres and artists.
 
-Artists I love:
+Here are some of my favourite picks:
 
 - Kate Bush
 - Son Lux
+- Vulfpeck
```
Comment (Oldzii) [inline content/non_programming.md:36]: +1

**Review by Oldzii — CHANGES_REQUESTED** [review]

File: content/about.md (hunk 1 above)
Comment (Wint3rmute) [reply content/about.md]: Fixed, please take a look!

File: content/about.md (hunk 2 above)
Comment (Wint3rmute) [reply content/about.md:33]: <3

File: content/about.md (hunk 4 above)
Comment (Wint3rmute) [reply content/about.md:53]: Fixed!

File: content/about.md (hunk 5 above)
Comment (Wint3rmute) [reply content/about.md:54]: This is kinda complicated, as the core idea resolves mostly around QA/smoke testing of the satellites themselves. Maybe `automated satellite testing` will get the message through better?

File: content/about.md (hunk 6 above)
Comment (Wint3rmute) [reply content/about.md:100]: fixed!

File: content/about.md (hunk 7 above)
Comment (Wint3rmute) [reply content/about.md:101]: Doesn't sound strange to me, I'll ask more people to settle this

File: content/non_programming.md (hunk 8 above)
Comment (Wint3rmute) [reply content/non_programming.md:36]: <3

File: content/about.md (hunk 3 above)
Comment (Wint3rmute) [reply content/about.md]: Changed to something less childish, please take a look

Comment (Oldzii) [reply content/about.md]: karr

File: content/about.md (hunk 5 above)
Comment (Oldzii) [reply content/about.md:54]: let it be

File: content/about.md (hunk 7 above)
Comment (Oldzii) [reply content/about.md:101]: fair

File: content/non_programming.md [hunk 9]
```diff
@@ -27,36 +27,31 @@ cover:
...
-
-Honorable mentions:
-
 - Bryan Ferry
-- Radiohead
-- 21 Pilots
+- Adam Neely
+
+I'm currently fascinated by electronic sound design, exploring various approaches to sound synthesis --
+programming and DSP knowledge proved surprisingly useful here. Hopefully I'll soon find more time to combine
```
Comment (Oldzii) [inline content/non_programming.md:46]: more time ;)

File: content/about.md (hunk 7 above)
Comment (Wint3rmute) [reply content/about.md]: Alright, got a confirmation from @nuczlab, changed the line

File: content/non_programming.md (hunk 9 above)
Comment (Wint3rmute) [reply content/non_programming.md:46]: https://youtu.be/iKOGgn4hMd4

**Pull request merged**